public_dropin_gpu_environments/triton_server/dr_requirements.txt

```diff
@@ -1,2 +1,140 @@
+#
+# This file is autogenerated by pip-compile with Python 3.9
+# by the following command:
+#
+#    pip-compile --index-url=https://artifactory.devinfra.drdev.io/artifactory/api/pypi/pypi/simple/ --no-emit-index-url --no-emit-trusted-host dr_requirements.in
+#
+argcomplete==1.11.1
+    # via datarobot-drum
+certifi==2024.2.2
+    # via requests
+charset-normalizer==3.3.2
+    # via requests
+click==8.1.7
+    # via flask
+datarobot==3.3.1
+    # via datarobot-drum
+datarobot-drum==1.10.20
+    # via -r dr_requirements.in
+docker==4.4.4
+    # via datarobot-drum
+flask==2.2.5
+    # via
+    #   datarobot-drum
+    #   flask-cors
+    #   mlpiper
+flask-cors==4.0.0
+    # via mlpiper
+future==1.0.0
+    # via mlpiper
+idna==3.6
+    # via requests
+importlib-metadata==7.1.0
+    # via flask
+itsdangerous==2.1.2
+    # via flask
+jinja2==3.1.3
+    # via
+    #   datarobot-drum
+    #   flask
+julia==0.5.7
+    # via datarobot-drum
+markupsafe==2.1.3
+    # via
+    #   datarobot-drum
+    #   jinja2
+    #   werkzeug
+memory-profiler==0.61.0
+    # via datarobot-drum
+mlpiper==2.6.0
+    # via datarobot-drum
+mypy-extensions==1.0.0
+    # via datarobot
+numpy==1.26.4
+    # via
+    #   datarobot
+    #   datarobot-drum
+    #   pandas
+    #   pyarrow
+    #   scipy
+packaging==24.0
+    # via datarobot-drum
+pandas==2.0.3
+    # via
+    #   datarobot
+    #   datarobot-drum
+pillow==10.2.0
+    # via datarobot-drum
+progress==1.6
+    # via datarobot-drum
+protobuf==5.26.1
+    # via -r dr_requirements.in
+psutil==5.9.8
+    # via
+    #   memory-profiler
+    #   mlpiper
+py4j==0.10.9.7
+    # via
+    #   datarobot-drum
+    #   mlpiper
+pyarrow==14.0.1
+    # via datarobot-drum
+python-dateutil==2.9.0.post0
+    # via
+    #   pandas
+    #   strictyaml
+pytz==2024.1
+    # via pandas
+pyyaml==6.0.1
+    # via
+    #   datarobot
+    #   datarobot-drum
 requests==2.31.0
-datarobot-drum==1.10.21
```

Comment:
I guess we are going back to v1.10.20?

Reply:
the current drum version is [v.1.10.21](https://github.com/datarobot/datarobot-user-models/blob/master/custom_model_runner/datarobot_drum/drum/description.py#L7).

I'll need to bump version and update dropin env afterwards.